sonarcloud/paging/paging.go

```diff
@@ -15,5 +15,5 @@ type Paging struct {
 
 // End returns whether the last page has been reached or not
 func (p *Paging) End() bool {
-	return p.PageIndex >= p.Total/p.PageSize
+	return float64(p.PageIndex) >= float64(p.Total)/float64(p.PageSize)
```

Comment:
Good catch! Instead of casting to `float64`, could you move `p.PageSize` to the other side of the comparison?

```suggestion
	return p.PageIndex * p.PageSize >= p.Total
```

Reply:
Done!